src/neo/SmartContract/Native/Oracle/OracleContract.cs

```diff
@@ -34,6 +35,51 @@ public sealed class OracleContract : NativeContract
         internal OracleContract()
         {
             Manifest.Features = ContractFeatures.HasStorage;
+
+            var events = new List<ContractEventDescriptor>(Manifest.Abi.Events)
+            {
+                new ContractEventDescriptor
+                {
+                    Name = "OracleRequest",
+                    Parameters = new ContractParameterDefinition[]
```

Comment:
Add `RequestContract`?

Reply:
It seems it's not very necessary, which is already triggered in the `RequestContract`. The `ScriptHash` of `NotifyEventArgs` is the hash of `RequestContract`.

Reply:
Isn't it triggered in the oracle contract?

Reply:
> Add `RequestContract`?

Got it.